CITATION.cff

```diff
@@ -0,0 +1,22 @@
+cff-version: 1.1.0
+message: If you use this software, please cite it as below.
+authors:
+  - family-names: Sterling
+    given-names: Jonathan
+    orcid: https://orcid.org/0000-0002-0585-5564
+    website: https://www.jonmsterling.com/
+  - name: Favonia
```

Comment:
The [CFF converter](https://github.com/citation-file-format/cff-converter-python) drops this. It seems that the only options are to use `given-names` or `alias`, or to adopt Favonia's suggested [LaTeX format](https://favonia.org/). @favonia?

Reply:
For consistency I prefer the LaTeX format `Kuen-Bang {Hou (Favonia)}`. (But it seems I can see "Favonia" on the GitHub web page?)